.gitmodules

```diff
@@ -9,3 +9,6 @@
 [submodule "external/rapidjson"]
 	path = external/rapidjson
 	url = https://github.com/Tencent/rapidjson
+[submodule "external/trezor-common"]
+	path = external/trezor-common
+	url = https://github.com/ph4r05/trezor-common.git
```

Comment:
This will have to be added as a module on the monero repo.

Reply:
Do you mean the `trezor-common` will be needed to be forked to the https://github.com/monero-project ?

Reply:
Yes, and I meant submodule.

Reply:
OK great. Once you fork `https://github.com/trezor/trezor-common.git` to the Monero account I let me know and I will change this. Thanks!